scripts/jenkins/jobs-obs/openstack-trackupstream.yaml

```diff
@@ -155,6 +157,8 @@
               "openstack-horizon-plugin-ironic-ui",
               "openstack-horizon-plugin-magnum-ui",
               "openstack-octavia",
+              "openstack-murano",
+              "python-muranoclient",
```

Comment:
OTOH all *client packages are listed in the top exclude - likely because there is only a master branch and if we keep tracking that for Mitaka it will break things sooner or later. So better move python-muranoclient there, too.